src/Components/Collections/CollectionsCarousel.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+import 'swiper/css/bundle';
+import 'swiper/css';
+import React, { useRef } from 'react';
+import propTypes from 'prop-types';
+import './Collection.css';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import SwiperCore, {
+  Navigation, A11y,
+} from 'swiper';
+
+// import AuthoredWorkCard from './AuthoredWorkCard';
+// import IllustratedWorkCard from './IllustratedWorkCard';
+import Collection from './Collection';
+
+// Authored and illustrated work components
+function CollectionsCarousel({
+  elementArray, slidesAtATime, prevArrow, nextArrow, widthPercent, spaceBetweenEntries,
+}) {
+  SwiperCore.use([Navigation, A11y]);
+  const navigationPrevRef = useRef(null);
+  const navigationNextRef = useRef(null);
+  // const swiperHook = useSwiper();
+  // const swiper = useSwiper();
+
+  return (
+
+    <div
+        // className="swiper-container"
+      style={{
+        display: 'flex', flexDirection: 'row', height: '350', width: `${widthPercent}%`,
+      }}
+    >
+      <div
+        className="carousel-button-prev"
+        style={{
+          display: 'flex', justifyContent: 'end', flexGrow: 1, alignItems: 'center', marginRight: '10',
+        }}
+      >
+        <button
+          ref={navigationPrevRef}
+          style={{
+            border: 'none', borderRadius: '9999px', background: 'none', color: '#D0D0D0',
+          }}
+          type="button"
+        >
+          <img src={prevArrow} alt="Left navigation arrow" />
+
+        </button>
+      </div>
+      <Swiper
+        style={{
+          zIndex: '0', marginLeft: 'auto', marginRight: 'auto', width: '90%',
+        }}
+          // loop
+          // createElements
+          // centeredSlides
+        //   centerInsufficientSlides
+        // width={745}
+        height={350}
+        //   autoHeight
+        on="true"
+        breakpoints={{
+          320: {
+            slidesPerView: 2,
+            spaceBetween: 20,
+          },
+          // when window width is >= 480px
+          480: {
+            slidesPerView: 3,
+            spaceBetween: 30,
+          },
+          // when window width is >= 640px
+          640: {
+            slidesPerView: slidesAtATime,
+            spaceBetween: spaceBetweenEntries,
+          },
+        }}
+        direction="horizontal"
+        navigation={{
+          prevEl: navigationPrevRef.current,
+          nextEl: navigationNextRef.current,
+        }}
+        onSlideChange={() => console.log('slide change')}
+        modules={[Navigation, A11y]}
+      >
+        {elementArray.map((element) => (
+          <div style={{ paddingTop: '20', paddingBottom: '20' }}>
```

Comment:
The error is still popping up, I think you need to give the parent divs the key! move the key={element.id} up to this div instead